WoWsShipBuilder.Web/Data/ShipBuildContainer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using WoWsShipBuilder.Core.BuildCreator;
+using WoWsShipBuilder.Core.DataContainers;
+using WoWsShipBuilder.DataStructures;
+
+namespace WoWsShipBuilder.Web.Data;
+
+public sealed record ShipBuildContainer(Ship Ship, Build? Build, Guid Id, IEnumerable<int>? ActivatedConsumableSlots, bool SpecialAbilityActive, ShipDataContainer? ShipDataContainer)
```

Comment:
May be redundant, but some users of this record may actually need the modifiers  instead of the whole calculated ShipDataContainer (one example is the future acceleration charts, modifiers are needed, not the whole ShipData). We should consider adding the list of modifier to this record.

Reply:
Resolved by commit 20958a10b5da471444961e93c78a60d93652399b